runtime/vm/jvminit.c

```diff
@@ -3268,7 +3268,7 @@ isEmpty(const char * str)
 {
 	BOOLEAN isEmpty = TRUE;
 	while('\0' != *str) {
-		if (0 == isspace((unsigned char) *str)) {
+		if (0 == OMR_ISSPACE((unsigned char) *str)) {
```

Comment:
I think it's time the cast was removed: if it's not needed in `redirector.c`, it's not needed here.